src/player.cpp

```diff
@@ -250,7 +250,6 @@ player::player()
         vitamin_levels[ v.first ] = 0;
     }
 
-    recalc_sight_limits();
```

Comment:
Can anyone tell me if this is going to be a problem? I removed it because this constructor is called before we even reach the main game menu, causing a segfault trying to `get_map` (because there's no map yet), and there doesn't seem to be any point in recalculating the character's sight limits so early. There are plenty of other recalc calls to make up for it long before an actual character is spawned in-game.

If this is safe to remove, I wonder if the folowing `calc_encumbrance()` could go away too.

Or, perhaps both could be gated by some check for whether the map has been initialized yet. I'm open to suggestions.

Reply:
Resolved in 5fa219e